Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -3747,6 +3749,19 @@ private ObservableList<ElementInfo> GetAllElementsFromPage(string path, List<eEl
             return foundElementsList;
         }
 
+        private HTMLElementInfo GetElementInfo(string path, HtmlNode htmlElemNode, Tuple<string, eElementType> elementTypeEnum, IWebElement elemObject)
+        {
+            HTMLElementInfo foundElemntInfo = new HTMLElementInfo();
+            foundElemntInfo.ElementType = elementTypeEnum.Item1;
```

Comment:
remove GetElementInfo() and put code back to calling function

Reply:
GetElementInfo() function removed !